library_management_cookiecutter/templates/home.html

```diff
@@ -0,0 +1,46 @@
+{% include 'base.html' %}
+<html>
+
+<head>
+    <style>
+        .home-background {
+            height: 100vh;
+            width: 100%;
+            overflow: hidden;
+            background: #468170;
+
+            
+        }
+
+        .home-background img {
+            max-height: 610px;
+            width: 100%;
+            object-fit: cover;
+            overflow: hidden;
+            /* padding: 2%; */
+            overflow-y: auto;
+            position: static;
+            background-color: rgb(89, 143, 102);
+        }
+
```

Comment:
in static folder make css and js folder and all related file to css and js should be there.